Back-End/.env

```diff
@@ -0,0 +1 @@
+DATABASE_URI=mongodb+srv://cyf:6Cd37enBZ2YsqfWj@cluster0.5kxur.mongodb.net/<dbname>?retryWrites=true&w=majority
```

Comment:
You should **not** check this data into GitHub - everyone in the world can now access our database 😱

As well as removing this file you need to rotate the credentials by resetting that user's password